sw/host/opentitanlib/src/app/config/hyperdebug_teacup.json

```diff
@@ -36,6 +36,7 @@
     },
     {
       "name": "IOA4",
+      "mode": "Input",
       "alias_of": "CN8_10"
     },
```

Comment:
Aside, is there any way we can give these semantic names in these configs?
Either through comments or `alias_of` etc?

Reply:
This pin is not used by hyperdebug during this test, but if it's not configured as input it would disturb the PMOD pins as they are shared.

Reply:
It is entirely possible to use `alias_of` to give a name such as `PMOD1_2`, which could maybe better explain why it needs to be configured as `Input`.

But this change causes trouble for ChromeOS, as we use `hyperdebug_teacup.json` in combination with further json files that declare how we want HyperDebug pins to default.   (And one of our files declares `IOA4` as `Alternate`.

I have created PR #23202, which separates this file into one that tries to merely declare physical routing of the Teacup board, which all first and third party users would want to include, and then a separate file with declarations such as this one, that OpenTitan test suites expect HyperDebug to have some default configuration of specific pins, which third party users would not include.  (Even in the OpenTitan repo, some other test suite might want to have HyperDebug drive CN8_10, in order to test something else.)